cookies-session-management-access-control/README.md

```diff
@@ -0,0 +1,344 @@
+Or can a non-admin user access and create API tokens that should only be created by an admin user?
+So, in order to test all the possible IDOR vulnerabilities, you should think like a hacker.
+
+You can provide permissions on your application for all endpoints.
+If your `privatesection` endpoint includes the API requests such as `/api/privatesection/admins`, `/api/privatesection/console`, `/api/privatesection/tokens`, you can block the endpoint for non-admin users.
+
+Moreover, to make the attacker’s job harder or prevent it altogether, you can use hash functions and hashed values instead of regular numbers and strings.
+
+## robots.txt: Preventing Caching
+
+The **robots.txt** file provides a list of instructions for automated Web crawlers, also called robots or bots.
+Standardized at [robotstxt](http://www.robotstxt.org/robotstxt.html), this file allows you to define, with a great deal of precision, which files and directories are off-limits to Web robots.
+The robots.txt file must be placed in the root of the Web server with permissions that allow the Web server to read the file.
+Lines in the file beginning with a # sign are considered comments and are ignored.
+Each line not beginning with a `#` should begin with either a User-agent or a disallow statement, followed by a colon and an optional space.
+These lines are written to disallow certain crawlers from accessing certain directories or files.
+
+Each Web crawler should send a user-agent field, which lists the name or type of the crawler.
+The value of Google’s user-agent field is `Googlebot`.
+To address a disallow to Google, the user-agent line should read:
+
+`User-agent: Googlebot`
+
+According to the original specification, the wildcard character `*` can be used in the user-agent field to indicate all crawlers.
+The disallow line describes what, exactly; the crawler should not look at.
+
+**NOTE:** Hackers don’t have to obey your robots.txt file.
+In fact, Web crawlers really don’t have to either, although most of the big-name Web crawlers will, if only for the “CYA” factor.
+One fairly common hacker trick is to view a site’s robots.txt file first to get an idea of how files and directories are mapped on the server.
+In fact a quick Google query can reveal lots of sites that have had their robots.txt files crawled.
+This, of course, is a misconfiguration, because the robots.txt file is meant to stay behind the scenes.
+
+## Sitemap.xml
+
+The sitemap.xml is a simple XML page which could be available on some websites and provide a “roadmap” for Google to the important pages that need to be crawled.
+It’s a SEO (Search Engine Optimization) tool to help with the visibility of your website on the internet, but it could also be useful for a hacker, serving basically the same purpose: to give him a roadmap to every page.
+
+## Examples of Google Dorking
+
+### Google Hacking tool from [Pentest-Tools](https://pentest-tools.com/information-gathering/google-hacking)
```

Comment:
I suggest putting these two in a bulleted list for more legibility

Reply:
Done.